search.go

```diff
@@ -16,15 +16,6 @@ type reqSearch struct {
 	} `json:"results"`
 }
 
-// SearchHosts searches through the hosts facet, returning matching hostnames.
-func (client *Client) SearchHosts(search string) ([]string, error) {
-	var out reqSearch
-	if err := client.doJsonRequest("GET", "/v1/search?q=hosts:"+search, nil, &out); err != nil {
-		return nil, err
-	}
-	return out.Results.Hosts, nil
-}
-
```

Comment:
Can we keep this function for backward compatibility and wrap the new search hosts function?